components/badge/bagde.css

```diff
@@ -0,0 +1,55 @@
+.badge-container {
```

Comment:
small mistake the file name is "badge.css" instead of "badge.css" here

Reply:
Thanks a lot for bringing that to my notice else I would've gone ahead with the wrong file name.